cfy_manager/networks/networks.py

```diff
@@ -20,17 +20,30 @@
 from ..logger import get_logger, setup_console_logger
 from ..constants import (
     NETWORKS_DIR,
+    CLOUDIFY_HOME_DIR,
+    VERBOSE_HELP_MSG,
 )
 from ..utils.certificates import (
     create_internal_certs,
     load_cert_metadata,
     store_cert_metadata
 )
+from ..config import config
+from ..components.service_names import MANAGER
+from ..components.components_constants import HOSTNAME
+from ..components.globals import _set_hostname
```

Comment:
It's a convention in python that if a function starts with `_`, it is an auxiliary function and should only be used in the specific module. So it's better not to import it if not necessary, or alternatively, change its name.

Reply:
well that set_hostname is somehow needed here, so we can change its original name but nothing prevent you from import it with `_`

Reply:
Correct, nothing is preventing us from importing it, but it's not following the convention. Anyway, I don't think you need this function - comment below.

Reply:
will move make generic then ;)